packages/office-ui-fabric-react/src/components/SearchBox/docs/SearchBoxBestPractices.md

```diff
@@ -1,18 +1,8 @@
-## Zero input state
+### Zero input state
 
 When the user has clicked into the SearchBox, but has not entered any text, there is an opportunity to display "hint text" within the input field, explaining what a user can do next. This could prompt a user to search for specific type content, or explain the scope of the search. Examples include "type to search", "try searching for \<x\>", "search for a place" or "type to search in \<x location\>".
 
-## Autocomplete suggestions
```

Comment:
good example of design component documentation the explains how you SHOULD implement auto complete, even if we haven't wired it into the react control.

Reply:
I'm swinging the other direction on this one. It's confusing users (per the issue), implying the `SearchBox` has autocomplete functionality. In fact, looking at `SearchBox` implementation it barely seems to add value over a regular `TextField`. I'm questioning the need for `SearchBox` entirely, per my other comment on this PR. I think this is a good example of where Fabric could narrow its scope a bit.